### ISSUE TITLE
"debug" is far too verbose

### ISSUE THREAD

**Issue by vendethiel** [dresende/node-orm2#642]
Hi,

I enabled the debug mode to get a hint at which query were runnings, but I started getting a spam of which packets were transmitted (`HandshakeInitializionPacket` then `ClientAuthenticationPacket` then `OkPacket`...)

My query string is the following:

```
mysql://root@localhost/bdiscuss?debug=true
```

my options:
```js
db.settings.set('instance.autoFetchLimit', 3)
db.settings.set('instance.cache', false)
```

**Comment by dxg** [comment]
This didn't use to happen. Maybe it's a mysql package 'feature'?
I'm not sure where it came from.

**Comment by dxg** [comment]
Fixed in v2.1.27

**Comment by vendethiel** [comment]
thank you!
